homeassistant/components/climate/nest.py

```diff
@@ -115,6 +116,18 @@ def current_operation(self):
             return STATE_UNKNOWN
 
     @property
+    def current_hvac_state(self):
```

Comment:
Good question. @technicalpickles, you're the Nest grandmaster 👍 what do you think?